### PR TITLE
More small overlay funcs

### PR DESCRIPTION
**Pull request by sozud**

### CODE REVIEW

File: src/overlays/ovl_i4/D4BD0.c [hunk 1]
```diff
@@ -154,7 +154,9 @@ void func_i4_8011B1E4(void) {
 
 #pragma GLOBAL_ASM("asm/us/rev0/nonmatchings/overlays/ovl_i4/D4BD0/func_i4_8011B380.s")
 
-#pragma GLOBAL_ASM("asm/us/rev0/nonmatchings/overlays/ovl_i4/D4BD0/func_i4_8011B3DC.s")
+Gfx* func_i4_8011B3DC(Gfx* arg0, unk_800E3A28* arg1) {
+    return arg0;
```
Comment (inspectredc) [inline src/overlays/ovl_i4/D4BD0.c:158]: Realise I probably missed this on the last review but general `Gfx* arg0` will be named as `Gfx* gfx`

```suggestion
Gfx* func_i4_8011B3DC(Gfx* gfx, unk_800E3A28* arg1) {
    return gfx;
```

File: src/overlays/ovl_i2/9BD00.c [hunk 2]
```diff
@@ -233,9 +233,15 @@ void func_i2_80100DDC(u8* arg0, s32 arg1, u16 arg2);
...
-#pragma GLOBAL_ASM("asm/us/rev0/nonmatchings/overlays/ovl_i2/9BD00/func_i2_8010356C.s")
+s32 func_i2_801034F8(s32, s32);
 
-#pragma GLOBAL_ASM("asm/us/rev0/nonmatchings/overlays/ovl_i2/9BD00/func_i2_8010358C.s")
+void func_i2_8010356C(s32 arg0) {
+    func_i2_801034F8(arg0, 0xE);
+}
+
+void func_i2_8010358C(s32 arg0) {
+    func_i2_801034F8(arg0 + 0x12, 0xE);
```
Comment (inspectredc) [inline src/overlays/ovl_i2/9BD00.c:243]: This looks like it's going to be &arg0->unk_12 but not sure if it's possible to figure out the struct from here

**Review by inspectredc — COMMENTED** [review]
Nothing blocking but would like to hear your thoughts on handling things

File: src/overlays/ovl_i2/9BD00.c (hunk 2 above)
Comment (sozud) [reply src/overlays/ovl_i2/9BD00.c:243]: Here's a function that uses this one https://decomp.me/scratch/qz4tm
It looks like it's passing in `extern u8 D_i2_8010D7F0[];`

Comment (inspectredc) [reply src/overlays/ovl_i2/9BD00.c:243]: ah alright, I think that D_i2_8010D7F0 is eventually gonna be an array of a struct of size 0x19E0 but I haven't quite figured out what yet so I'm happy to leave this as is, thanks for looking into it!

**Pull request merged**